TIL/topics/dart_basics/Result 패턴.md

```
@@ -0,0 +1,46 @@
+---
+
+---
+---
+### 런터임 오류뿐만 아니라 비즈니스 로직의 잘못된 결과 반환에 대해서도 실패 처리를 해줘야한다.
+- sealed class를 활용한 result 패턴
+	```dart
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)